src/vivarium/testing_utilities.py

```diff
@@ -7,6 +7,7 @@
 
 """
 
+from itertools import product
```

Comment:
Why is this file packaged instead of living under the tests module?

Reply:
I don't understand your question.

Reply:
Because it is used by VPH. When `vivarium_testing_utils` is more developed, it should probably move there.